Python/README.md

```diff
@@ -178,98 +115,78 @@
 - [Sieve of Eratosthenes](math/prime_number_using_sieve_of_Eratosthenes.py)
 - [Fibonacci number](math/Fibonacci.py)
+- [Fibonacci number with Recursion](math/fibonacci_recursion.py)
+- [Check Armstrong Number](math/CheckArmstrongNumber.py)
 - [Strassen's Matrix Multiplication](math/Strassen's_Matrix_Mul.py)
+- [Ackermann Function](math/ackermann_function.py)
+- [Ugly Number](math/UglyNumber.py)
 - [Prime Number Iterative and Recursive](math/prime_number.py)
 - [Catalan Number](math/Catalan_Number.py)
 
-## Other
-* [Check Inversion](other/Check_Inversion.py)
-* [Kadane's Algorithm](other/Kadanes_Algorithm.py)
-* [Longest substring without repeating character](other/longest_Substring.py)
-* [Ocurrence of a number using binary search](other/find_freq_using_bs.py)
-* [Chinese Remainder theoram](other/chinese_remainder_theorem.py)
-* [Reverse a string using stack](other/Reverse_string_using_stack.py)
-* [Satellite Image Analysis using numpy](other/Satellite_Image_Analysis_using_numpy.ipynb)
-* [Sudoku Solver](other/sudoku_backtracking.py)
-* [Moore's Algorithm](other/Moore_Algorithm.py)
-- [Ugly Number](math/UglyNumber.py)
-- [Extended Euclidean Algorithm](math/Extended_Euclidean_Algorithm.py)
-- [Least Common Multiple](math/Least_common_multiple.py)
-- [Double Factorial](math/double_factorial.py)
-- [Euler Totient Function](math/euler_totient_function.py)
-
-## Backtracking
-
-- [N Queens Problem](backtracking/N_Queens.py)
-- [Knight's Tour](backtracking/Knight's_Tour.py)
-- [Hamiltonian Cycle](backtracking/Hamiltonian_Cycle.py)
-- [Sudoku Solver](backtracking/sudokuSolver.py)
-- [Subset Sum](backtracking/Subset_Sum.py)
- 
-## Dynamic Programming
-
-- [Coin Change Problem](dp/CoinChange.py)
-- [Longest Common Subsequence](dp/Longest_Common_Subsequence.py)
-- [Longest Increasing Subsequence](dp/Longest_Increasing_Subsequence.py)
-- [Binary Trees with N nodes](dp/binary_trees_with_n_nodes.py)
-- [Unique BST with N nodes](dp/Unique_BST.py)
-- [Edit Distance](dp/EditDistance.py)
-- [Knapsack Unbounded](dp/Knapsack_Unbounded.py)
-- [Ugly Number](dp/Ugly_Number.py)
-
 
 ## Blockchain
 
 _add list here_
 
+
 ## Other
 
 - [Check Inversion](other/Check_Inversion.py)
 - [Distinct Elements Count](other/Distinct_elements_count.py)
 - [Find frequency using Binary Search](other/find_freq_using_bs.py)
 - [Kadane's Algorithm](other/Kadanes_Algorithm.py)
-- [Least Common multiple](other/Least_common_multiple.py)
 - [Longest Substring](other/longest_Substring.py)
 - [Merging Sorted arrays](other/merge_sorted_arrays.py)
 - [Maximum Square Matrix with all 1's](other/Maximum_Square_Matrix.py)
 - [Possible Code Recursion](other/Possible_code_recursion.py)
 - [Reverse a string using stack](other/Reverse_string_using_stack.py)
 - [Russian Peasant Algorithm](other/Russian_Peasant_Algorithm.py)
+- [Ocurrence of a number using binary search](other/find_freq_using_bs.py)
+- [Chinese Remainder theoram](other/chinese_remainder_theorem.py)
```

Comment:
```suggestion
- [Chinese Remainder theorem](other/chinese_remainder_theorem.py)
```

Reply:
Okay I will change the location of the Chinese Remainder Theorem to maths folder and add the description to it as i use to do in all my PRs.